app/views/components/check_uk_visa/_result-card.html.erb

```diff
@@ -0,0 +1,60 @@
+<%
+  type ||= nil
+  title ||= nil
+  description ||= nil
+  attributes ||= []
+  url ||= nil
```

Comment:
Minor: we should remove the default values if we are requiring values to be present.

Reply:
Thanks Pete, I'll leave them in for now but sounds good to me to update the view level validation to tests (and remove the default values whilst doing so).